app/controllers/api/v1/activities_controller.rb

```diff
@@ -6,6 +6,9 @@ def latest
 
   def just_updated
     versions = Version.just_updated(50)
+    if params[:since].present?
```

Comment:
Is a test for `presence` enough? Won't this blow up for weird `params[:since]`? It would nice to [document what format is expected](http://guides.rubygems.org/rubygems-org-api/#get---apiv1activityjust_updated).